### DISCUSSION TITLE
colmap2nerf.py - ERROR 'input_path' is not a directory

### DISCUSSION THREAD

**Discussion by R-Ladig** [NVlabs/instant-ngp#492] [Q&A]
When trying to prepare my own NeRF dataset and using the colmap2nerf.py script with either a video or an image dataset, I get this during the loading of the database (after exhaustive feature matching):
 ==== running: colmap bundle_adjuster --input_path colmap_sparse/0 --output_path colmap_sparse/0 --BundleAdjustment.refine_principal_point 1

ERROR: `input_path` is not a directory
FATAL: command failed

Any idea how to fix this?

**Comment by sidk99** [comment]
Any updates?

**Reply by R-Ladig** [reply]
So ehm... Found a "solution". Comap straight up doesn't work if you download the precompiled version on Ubuntu 20.04 LTS... Compiled the newest version on master and is now working flawlessly.

**Reply by senyangjiang** [reply]
Can confirm. Need to compile from source in order for colmap to work on Ubuntu 20.04

**Reply by Lokesh-26** [reply]
> So ehm... Found a "solution". Comap straight up doesn't work if you download the precompiled version on Ubuntu 20.04 LTS... Compiled the newest version on master and is now working flawlessly.

Hello,
Can you explain it more on solving the error. When I run the ns-process data for images, I tried with 4 datasets. only with one of the dataset it causing me this error. Other datasets are working fine.

**Comment by R-Ladig** [comment]
Nope.

It's clear that the colmap_sparse/0 folder is not beeing generated but there are a ton of image features found in the step before that and they should be clear enough to do some matching.
